eos/form-factors/analytic-b-to-gamma-qcdf.cc

```diff
@@ -757,7 +757,11 @@ namespace eos
         };
         c = c + c_c_star;
 
-        auto [a_begin, a_end] = blcdas->coefficient_range(mu);
+        // Workaround for Clang: cannot presently capture structured bindings
+        // auto [a_begin, a_end] = blcdas->coefficient_range(mu);
+        auto a_range = blcdas->coefficient_range(mu);
```

Comment:
This is weird. Clang supports structured bindings since version 8.

Reply:
Indeed. The problem is subtle: capturing variables is well defined since lambdas were introduced. Then came bindings, which are technically not variables. Capturing bindings was later also well defined by the C++ standard, but never implemented by Clang.